src/inverseRobot.js

```diff
@@ -8,6 +8,17 @@
 
 function inverseRobot(robot) {
   // write code here
+  const newRobot = {};
+
+  for (const key in robot) {
+    if (newRobot[robot[key]] !== undefined) {
+      return null;
+    }
+
+    newRobot[robot[key]] = key;
```

Comment:
You need to create a variable for robot[key]

Reply:
agree